_posts/2019-02-06-Age-Targeting-Part2.md

```diff
@@ -0,0 +1,253 @@
+---
+layout: post
+title: Increase your target audience via ML on Spark [part 2]
+subtitle: Preprocessing, train and predict
+category: data
+tags: [adtech, howto, spark, ml]
+author: Alessio Borgheresi
+header-img: "images/Age-Targeting/age_targeting2.jpeg"
+---
+
+In mobile advertising it is possible to target specific audiences based on some characteristics of the users, such as the age. 
+However, this information is not available for every users.
+We build a supervised machine learning model to extend the knowledge over the whole dataset.
+In the previous [post](https://adello.github.io/Age-Targeting-Part1/) we described how the reliable ground truth dataset was created.
+It looks like this:
+<p align="center"> <img src="../images/Age-Targeting/table_ground_truth_age.png"> </p>
+For these devices we know the age and we convert it into 5 ranges: 13_17, 18_24, 25_34, 35_54, 55+.
+These are the classes used to train the machine learning model.
```

Comment:
These are the classes used as target in the training of the machine learning model.